dispatch/apps/api/serializers.py

```diff
@@ -38,6 +28,54 @@ class Meta:
             'image'
         )
 
+class UserSerializer(DispatchModelSerializer):
+    """
+    Serializes the User model.
+    """
+
+    email = serializers.EmailField(
+        required=True,
+        validators=[UniqueValidator(queryset=User.objects.all())]
```

Comment:
is this required, on top of the [unique db constraint](https://github.com/ubyssey/dispatch/blob/develop/dispatch/apps/core/models.py#L26) on the User model itself?